### ISSUE TITLE
Adding three new apps to the app store

### ISSUE THREAD

**Issue by pstan26** [blockstack/blockstack-browser#1065]
Hi @guylepage3 starting this task so you are able to apply the rounded edged to them. If there are instructions we can give to apps in the future so you don't have to do this everytime, I'd love to get people using it.

Below is a list so you can check them off as the you round them and they get added to the next build. Suggest added the first 3 listed here:

**CoinStack**
- [x] Rounded corners for logo
- [x] Added URL & Logo to home screen
http://www.coinstack.one/
![coinstack](https://user-images.githubusercontent.com/5605552/32709170-614e69de-c7fc-11e7-8a7c-f56f7633f057.png)

**Coins**
- [x] Rounded corners for logo
- [x] Added URL & Logo to home screen
https://use.coinsapp.co/
![coins](https://slack-imgs.com/?c=1&url=https%3A%2F%2Fd3n6sqn01en2u4.cloudfront.net%2Fineffableco%252Fc892a001-545e-4791-80d4-3dc4bd893d65_coinsapp.png%3Fw%3D512)

**Coinfolio**
- [x] Rounded corners for logo
- [x] Added URL & Logo to home screen
http://coinfolio-app.herokuapp.com/
![profile_picture_symbol](https://user-images.githubusercontent.com/5605552/32711923-913080a2-c80f-11e7-9d13-d90f07b509c2.png)

-----------------------------------------------------------------
**Blockportfolio**
- [x] Rounded corners for logo
- [ ] Added URL & Logo to home screen 
https://www.blockportfol.io/
![blockportfolio_logo](https://user-images.githubusercontent.com/5605552/32708840-798423b0-c7fa-11e7-8569-332e6d3c107d.png)

**Coindex**
- [ ] Rounded corners for logo
- [ ] Added URL & Logo to home screen
https://coindexapp.herokuapp.com/
*needs logo*

**Saifu**
- [x] Rounded corners for logo
- [ ] Added URL & Logo to home screen
http://104.236.62.99:5000/
![icon-512x512](https://user-images.githubusercontent.com/5605552/32709312-6a7983c6-c7fd-11e7-812a-dc698cf27df1.png)

**Portfo**
- [x] Rounded corners for logo
- [ ] Added URL & Logo to home screen
http://portfo-blockstack.herokuapp.com/
![favicon](https://user-images.githubusercontent.com/5605552/32708924-0f26fb22-c7fb-11e7-85ee-1b8fc362e965.png)

adding more right now... but once they are all uploaded and we've gotten everything ready, @yknl and @larrysalibra will add these to the new build.

**Comment by guylepage3** [comment]
@pstan26 can we get a version of the "Coins" icon that does not have the curves already applied to the icon as we have our own custom rounded corners. Thank you.

**Comment by pstan26** [comment]
Since this is high priority to complete this morning and these folks are in SF, I think its best if, for now, we zoom in and make our own. I wouldn't wait on those fellas.

**Comment by guylepage3** [comment]
@pstan26 sure. I can fill in the space in Photoshop. What's the timing for this?

**Comment by guylepage3** [comment]
Working on this now.

**Comment by guylepage3** [comment]
![coins-icon](https://user-images.githubusercontent.com/1711854/32732078-848a8baa-c859-11e7-926c-5f1330c4273b.png)

**Comment by guylepage3** [comment]
@pstan26 want me to drum up a quick logo for Coindex?

**Comment by guylepage3** [comment]
@pstan26 incorrect URL for "coins" just updated to.. http://use.coinsapp.co/

**Comment by guylepage3** [comment]
@pstan26 when will we be adding the other app icons to the list?

**Comment by dantrevino** [comment]
Everyone had nifty urls so i got jealous: http://saifu.programmerscloset.org/

**Comment by guylepage3** [comment]
@pstan26 updated "Coinstack A" to "Coinstack" and removing the Blockportfolio app.

**Comment by patrickdevivo** [comment]
Any chance this is in order of submission time...?

**Comment by benrogmans** [comment]
@guylepage3 Hey Guy, any chance you could update "Coinstack" to "CoinStack"?

**Comment by guylepage3** [comment]
hey @benrogmans. Is that for "CoinStack" or "CoinStack B"? See above.. https://github.com/blockstack/blockstack-browser/issues/1065#issue-273301929

**Comment by benrogmans** [comment]
@guylepage3 CoinStack A with the 'CS' logo 😁

**Comment by guylepage3** [comment]
Will do sir. 👍

**Comment by benrogmans** [comment]
Thanks!

**Comment by guylepage3** [comment]
@benrogmans. Updated and will be in the next build. :)

**Comment by patrickdevivo** [comment]
@guylepage3 please see this PR: https://github.com/blockstack/blockstack-browser/pull/1069 for a commit addressing the icon/text for BlockPortfolio

**Comment by xanbots** [comment]
Coinstack B (From Satraj) is working and ready to add to the browser.

**Comment by pstan26** [comment]
Satraj is going to suggest a new name as well.

**Comment by pstan26** [comment]
some options for Satraj: Coinfolio, tokenfolio, cointrack, tokendex, coinlog, coinbook, tokenbook, coinfile, coinfund, tokencount, coinsheet, tokensheet

Looks like he is selecting "Coinfolio"

**Comment by pstan26** [comment]
@guylepage3 @larrysalibra @yknl   Coinfolio is now ready to be added to the browser under "user-ready apps" (I edited the name from Coinstack B to Coinfolio)
 :-)
Looking forwadr to seeing it in the next build

**Comment by satbam** [comment]
Thanks @pstan26

**Comment by guylepage3** [comment]
Can you remind me about this issue this afternoon @pstan26 ?

**Comment by xanbots** [comment]
Hi @guylepage3 just sending a reminder as @pstan26 is focused on investor outrweach

**Comment by satbam** [comment]
hi @xanbots @guylepage3 just wanted to checkin to see if Coinfolio was added to the browser?
http://coinfolio-app.herokuapp.com/

**Comment by guylepage3** [comment]
@satbam. not yet. next build is on Monday afternoon.

**Comment by guylepage3** [comment]
closing this issue. @xanbots @satbam can one of you open a new issue for coinfolio app icon? thanks.

**Comment by guylepage3** [comment]
I created the new issue here.. https://github.com/blockstack/blockstack-browser/issues/1089
